frontend/src/types/authorship.ts

```diff
@@ -0,0 +1,11 @@
+export enum FilesSortType {
+  LineOfCode = 'lineOfCode',
```

Comment:
Just a grammar check. Will it be more suitable to use the plural form `lines` instead

Reply:
I agree with this, I'll change it to `lines`. It's also similar [in c-zoom](https://github.com/reposense/RepoSense/pull/1965/files#diff-f1f207cd3bfc1e4ba3264657c296cc74bdf556265c4879b07ca82d032f9a86c9R3), should we change it in #1965 too?

Reply:
yeah sure. Please go ahead

Reply:
@ckcherry23 thanks!